rest/komand_rest/util/util.py

```diff
@@ -138,7 +142,7 @@ def call_api(
             if response.status_code >= 500:
                 raise PluginException(preset=PluginException.Preset.SERVER_ERROR, data=response.text)
 
-            if 200 <= response.status_code < 300:
+            if 200 <= response.status_code and response.status_code < 300:
```

Comment:
For some reason, this line failed on my machine. I'm running py 3.8. 

Splitting it was the only way I could get it to work.